(ForumsTeam)Requesting to update Hyper Link

v-viksur opened the pull request:
In this documentation under the section "Choose a VM restore configuration" the hyper link "Restore a VM with special network configurations" is not pointing to the correct section. The hyper link is pointing to the top of the documentation. It should point to the correct section.
Prasent Hyper Link: https://docs.microsoft.com/en-us/azure/backup/backup-azure-arm-restore-vms#restore-a vm-with-special-network-configurations 
Correct Hyper Link : https://docs.microsoft.com/en-us/azure/backup/backup-azure-arm-restore-vms#restore-vms-with-special-network-configurations

azurecla commented:
Hi __@ruchic__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

v-viksur committed:
(ForumsTeam)Requesting to update Hyper Link

In this documentation under the section "Choose a VM restore configuration" the hyper link "Restore a VM with special network configurations" is not pointing to the correct section. The hyper link is pointing to the top of the documentation. It should point to the correct section.
Prasent Hyper Link: https://docs.microsoft.com/en-us/azure/backup/backup-azure-arm-restore-vms#restore-a vm-with-special-network-configurations 
Correct Hyper Link : https://docs.microsoft.com/en-us/azure/backup/backup-azure-arm-restore-vms#restore-vms-with-special-network-configurations
